content/docs/ref/runner.md

```diff
@@ -52,9 +52,10 @@ Any [generic option](/doc/ref) in addition to:
   purposes.
 - `--cloud-aws-security-group=<...>`:
   [AWS security group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html)
-  name.
-- `--cloud-aws-subnet-id=<...>`:
-  [AWS subnet](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#subnet-basics).
+  name or ID.
```

Comment:
```suggestion
  identifier.
```

While security group names asre also unique, I would prefer to deprecate that option and encourage users to use identifiers instead.